src/components/pages/Volunteers/AddContactModal.tsx

```diff
@@ -0,0 +1,410 @@
+import React, { useState } from 'react';
+import { Button, Form, Modal, Col } from 'react-bootstrap';
+import { submitButton } from 'src/redux/modules/volunteerContacts';
+import { connect, ConnectedProps } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { RootState } from '../../../redux';
+
+interface AddContactModalProps {
+  volunteer: Volunteer;
+  show: boolean;
+  handleClose: () => void;
+  token: string;
```

Comment:
would love to hear ur thoughts on this one as well. Think we could do some refactoring here on a fast-follow PR. 
It seems to be bad code design to have to pass down token everywhere. I was researching some better ways to do I: https://michaelwashburnjr.com/2017/11/21/best-way-to-store-tokens-redux/

Would love to hear ur thoughts